rpc/wrtc_call_queue_mongodb.go

```diff
@@ -427,7 +423,7 @@ func (queue *mongoDBWebRTCCallQueue) changeStreamManager() {
 	for {
 		// Note(erd): this could use condition variables instead in order to be efficient about
 		// change stream restarts, but it does not feel worth the complexity right now :o)
-		if !utils.SelectContextOrWaitChan(queue.cancelCtx, ticker.C) {
+		if !utils.SelectContextOrWaitChan(queue.activeStoppableWorkers.Context(), ticker.C) {
```

Comment:
Same remark about `changeStreamManager` being a function that accepts a context.